dummy/dialog.html

```diff
@@ -0,0 +1,27 @@
+<meta charset="UTF-8" />
+<script>
+  UPLOADCARE_PUBLIC_KEY = 'demopublickey'
+  UPLOADCARE_TABS = 'default huddle'
+  UPLOADCARE_DEBUG_UPLOADS = true
+</script>
+<script src="./uploadcare.full.js"></script>
```

Comment:
as I remember we don't use full version more

Reply:
Yes